.env

```diff
@@ -37,4 +37,8 @@ ILIOS_ELASTICSEARCH_UPLOAD_LIMIT=8000000
 # For Gmail as a transport, use: "gmail://username:password@localhost"
 # For a generic SMTP server, use: "smtp://localhost:25?encryption=&auth_mode="
 # Delivery is disabled by default via "null://localhost"
-ILIOS_MAILER_URL=null://localhost
\ No newline at end of file
+ILIOS_MAILER_URL=null://localhost
+
+# Controls whether application config should be cached in memory for the duration of any given PHP process, or not.
+# The default is TRUE.
+ILIOS_ENABLE_APPLICATION_CONFIG_CACHE=true
```

Comment:
Given that this will only be used within tests I'd advocate for it not to be a ENV variable and just live in a parameter in https://github.com/ilios/ilios/blob/master/config/services_test.yaml

Reply:
agreed. i changed this as suggested.